go.mod

```diff
@@ -1,11 +1,10 @@
 module github.com/gametimesf/testy
 
-go 1.18
+go 1.21.0
```

Comment:
you can specify patch versions in go.mod now? Or should this be `1.21`

Reply:
> Go 1.21 introduces a small change to the numbering of releases. In the past, we used Go 1.N to refer to both the overall Go language version and release family as well as the first release in that family. Starting in Go 1.21, the first release is now Go 1.N.0. Today we are releasing both the Go 1.21 language and its initial implementation, the Go 1.21.0 release. These notes refer to “Go 1.21”; tools like go version will report “go1.21.0” (until you upgrade to Go 1.21.1). See “[Go versions](https://tip.golang.org/doc/toolchain#versions)” in the “Go Toolchains” documentation for details about the new version numbering.

https://tip.golang.org/doc/go1.21

Reply:
hmm don't know how I feel about that. It feels like unnecessarily added complexity in the name of not having to go download the latest go version on your own.